### PR TITLE
New folk jpg

### PR DESCRIPTION
**Pull request by guushoekman**
- Added people
- Photos in jpg

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -63,14 +63,16 @@
...
+          <li>{% include team.html name="Lenina Rassool" title="Communications" twitter="@Nina_210" image_class="ch-img-lenina" %}</li>	
+          <li>{% include team.html name="Guus Hoekman" title="Wrangling Fellow <br> " image_class="ch-img-guus" %}</li>		
+	  <li>{% include team.html name="Jeanette Strydom" title="Community Manager" twitter="@jenstrydom" image_class="ch-img-jeanette" %}</li>
 	
 
           <h2 class="dark-grey">Alumni</h2>
           <li>{% include team.html name="Osman Siddiqi" title="Visiting Fellow<br/>MPA-ID Candidate<br/>Harvard Kennedy School" image_class="ch-img-osman" %}</li>
-          <li>{% include team.html name="Khutjo Konyana" title="Data Herder" twitter="@SwiftieDarlinG" image_class="ch-img-khutjo" %}</li>
-          <li>{% include team.html name="Petrus J.v.Rensburg" title="Developer" twitter="@petrusjvr" image_class="ch-img-petrus" %}</li>
+          <li>{% include team.html name="Khutjo Konyana" title="Data Herder" twitter="@SwiftieDarlinG <br></br>" image_class="ch-img-khutjo" %}</li>
```
Comment (longhotsummer) [inline about.html]: What's the br stuff for?

If it's necessary, it should be `<br>` or `<br><br>`. The tag doesn't need to be closed.

Comment (guushoekman) [inline about.html]: It's to align the photos. Some people had four lines so others' photos weren't on the same level.

`<br>` didn't work, I'll change it to `<br><br>` now

**Comment by nefiger** [comment]
@longhotsummer you happy if we merge now?

**Comment by longhotsummer** [comment]
does it look ok? I'm only looking at the code.

**Pull request merged**